run_tests.sh

```diff
@@ -23,19 +23,5 @@ env GORACE="halt_on_error=1" go test -race ./...
 # golangci-lint (github.com/golangci/golangci-lint) is used to run each each
 # static checker.
 
-# set output format for linter
-if [[ -v CI ]]; then
-    OUT_FORMAT="github-actions"
-else
-    OUT_FORMAT="colored-line-number"
```

Comment:
These didn't make it to the yml file.  I also think you can specify both.

Reply:
@jholdstock says golangci-lint does a good  job at autodetection.